chess/engine.py

```diff
@@ -2772,6 +2785,9 @@ def ping(self) -> None:
             future = asyncio.run_coroutine_threadsafe(coro, self.protocol.loop)
         return future.result()
 
+    def draw(self) -> None:
+        self.protocol.draw()
```

Comment:
Simple function, but still needs to check that the engine is alive and be called from the event loop, much like https://github.com/niklasf/python-chess/blob/2379c55bcadd6ef9f2acb2e2f7f2476094be5ec1/chess/engine.py#L2892-L2894